WordPress/Sniffs/XSS/EscapeOutputSniff.php

```diff
@@ -208,6 +208,11 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
 
 			if ( T_OPEN_PARENTHESIS === $this->tokens[ $i ]['code'] ) {
 
+				if ( ! isset( $this->tokens[ $i ]['parenthesis_closer'] ) ) {
+					// Live coding or parse error.
+					break;
```

Comment:
Yep, that makes sense. So I guess for consistency we should change [the one place](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/51eb33b15f3aad5d74e1c816b5d8edc42163de25/WordPress/Sniffs/PHP/StrictInArraySniff.php#L103) we give an error for syntax issues. But that can be a separate PR, I guess.